app/stac_api/managers.py

```diff
@@ -41,8 +41,8 @@ def filter_by_bbox(self, bbox):
             logger.debug('Query parameter bbox = %s', bbox)
             list_bbox_values = bbox.split(',')
             if (
-                list_bbox_values[0] == list_bbox_values[2] and
-                list_bbox_values[1] == list_bbox_values[3]
+                float(list_bbox_values[0]) == float(list_bbox_values[2]) and
+                float(list_bbox_values[1]) == float(list_bbox_values[3])
```

Comment:
Float comparisons are a bit tricky, I think (see: https://stackoverflow.com/questions/26091689/python-floating-point-number-comparison)
We have our `float_in()` in `collection_summaries.py` for similar reasons.

I am not sure, if we would run into problems here, but it might be safer, to use a function for that comparison for equality.
E.g. something like this:
```
def floats_equal(float1, float2):
     if abs(float1 - float2) < whatever_small_number_e_g_1e-10:
         return True
    return False
```

Reply:
Even better :nerd_face:

Reply:
Oops forget about testing string equality, I didn't thought of this use case : `[6.10,47.10,6.1,47.1]` which would not pass the string equality but would pass the float equality !

Reply:
Haha, same thought at the same time :-)

Reply:
It is more about the POST, where 
{ [6.10,
      47.10,
  6.1,
       47.1]

brought the comparison into trouble